pallets/rbac/src/tests.rs

```diff
@@ -833,13 +833,26 @@ fn get_role_users_len_should_work() {
 }
 
 #[test]
-fn has_any_role_should_work() {
+fn does_user_have_any_role_in_scope_should_work() {
```

Comment:
Can a test be created where a user belongs to multiple scopes? To ensure that the iter_prefix returns a vector of vectors with the primary keys provided.